app/jobs/submit_iplc_listener_job.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+##
+# Background job to send requests to IPLC.
+# If Reshare has the item and we succesfully request it, let the user know it is on the way.
+# If Reshare does not have it or we cannot succesfully request it, trigger the normal Symphony request workflow.
+class SubmitIplcListenerJob < ApplicationJob
+  discard_on ActiveRecord::RecordNotFound do |job, _error|
+    Honeybadger.notify(
+      "Attempted to call Reshare for Request with ID #{job.request_id}, but no such Request was found."
+    )
+  end
+
+  def perform(request_id, instance_uuid)
+    request = Request.find(request_id)
+
+    Sidekiq.logger.info("Started SubmitIplcRequestJob for request #{request_id}")
+
+    begin
+      make_iplc_request(request, instance_uuid)
+    rescue StandardError => e
+      Honeybadger.notify("IPLC Request failed for #{request_id} with #{e}. Submitted to Symphony instead.")
+
+      request.send_to_symphony_now!
+    end
+
+    Sidekiq.logger.info("Completed SubmitIplcRequestJob for request #{request_id}")
+  end
+
+  def make_iplc_request(request, instance_uuid)
+    iplc_response = IplcWrapper.new(request, instance_uuid)
+
+    raise 'IPLC HTTP request failed' unless iplc_response.success?
+
+    request.borrow_direct_response_data = iplc_response.as_json
+    request.via_borrow_direct = true
+    # We don't care if this save was successful or not, because if it went through the back-end service, then requests is no longer the source of truth for this.
```

Comment:
Make rubocop happy
```suggestion
    # We don't care if this save was successful or not, because if it went through the back-end service,
    # then requests is no longer the source of truth for this.
```